src/writer/tests.py

```diff
@@ -0,0 +1,21 @@
+def invoke_event_handler(event_handler, **kwargs):
+    """
+    Invokes an event handler in the same way that streamsync would.
```

Comment:
Please change streamsync to Writer Framework

Reply:
this code was not necessary. I removed it